tests/lib/tools/boot-state

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: bootenv show [var]"
+    echo "       bootenv set <var> <value>"
+    echo "       bootenv unset <var>"
+    echo "       boot-path"
+    echo "       wait-core-post-boot"
+    echo ""
+    echo "Get information and manages the boot loader for the current system"
+    echo ""
+    echo "COMMANDS:"
+    echo "  bootenv show:         prints the whole bootenv or just the variable passed as parameter"
+    echo "  bootenv set:          sets the given var and value on boot configuration"
+    echo "  bootenv unset:        unsets the given var from boot configuration"
+    echo "  boot-path:            prints the boot path"
+    echo "  wait-core-post-boot:  waits until the snap_mode bootenv var is empty"
+}
+
+get_grub_editenv() {
```

Comment:
May I suggest to simplify this to `run_grub_editenv` which executes the rigth command with "$@". This way usage will be more natural, and if the command is missing, the failure more direct.

Reply:
done